### ISSUE TITLE
Why not use the scheduler to update learning rate ??

### ISSUE THREAD

**Issue by zhijiejia** [stevehuanghe/SCA-CNN#10]
Hey, I don't find the scheduler to update the learning rate in the training code? May add it can better?
